### PR TITLE
added comment parsing for profile.py

### PR DESCRIPTION
**Pull request by PickmanSec**
Replace comments inside of config file before execution to allow people to use profile.py without editing conf file.

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

**Comment by theopolis** [comment]
ok to test

**Comment by theopolis** [comment]
Issue/discussion reference: https://github.com/facebook/osquery/issues/1563

File: tools/tests/utils.py [hunk 1]
```diff
@@ -76,6 +79,17 @@ def queries_from_config(config_path):
     if "schedule" in config:
         for name, details in config["schedule"].iteritems():
             queries[name] = details["query"]
+    if "packs" in config:
+        for keys,values in config["packs"].iteritems():
+            with open(values) as fp:
+                packfile = fp.read()
+                #packcontent = rmcomment.sub('',packfile)
```
Comment (theopolis) [inline tools/tests/utils.py]: Why not apply the same comment scrubbing as with the main config?

Comment (PickmanSec) [inline tools/tests/utils.py]: k.

**Comment by PickmanSec** [comment]
I am not sure why this failed.

**Comment by theopolis** [comment]
Nice, can you merge the 3 commits into a single for this pull request: `git rebase -i HEAD~3` then change the `pick` to `s` for squash for the second and third lines? Finally `git push -f` to update this PR.

**Comment by PickmanSec** [comment]
Did those things.

**Comment by theopolis** [comment]
ok to test

**Pull request merged**